app/cinema/hall.py

```diff
@@ -0,0 +1,19 @@
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(self,
+                      movie_name: str,
+                      customers: list,
+                      cleaning_staff: "Cleaner"
+                      ) -> None:
```

Comment:
```suggestion
    def movie_session(self,
                      movie_name: str,
                      customers: list,
                      cleaning_staff: "Cleaner") -> None:
```

brackets must be located in one of two options: open and start with the text or have a line break between the text

Reply:
If this line becomes too long, choose another option